<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimestampAnnotationFormatWidget</class>
 <widget class="QWidget" name="TimestampAnnotationFormatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timestamp Format</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="l1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Select Timestamp Format</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="pTimeBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="pShowMs">
        <property name="toolTip">
         <string>Toggle Milli Seconds Display</string>
        </property>
        <property name="text">
         <string>Show millseconds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="pShowDate">
        <property name="toolTip">
         <string>Toggle Date Display</string>
        </property>
        <property name="text">
         <string>Show Date</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
